crates/swc_ecma_minifier/tests/compress/fixture/next/feedback-util-promisify/chunks/pages/_app-72ad41192608e93a/output.js

```diff
@@ -1997,7 +1997,7 @@
                         }
                         function stylizeWithColor(r, t) {
                             var e = inspect.styles[t];
-                            return e ? "\x1b[" + inspect.colors[e][0] + "m" + r + "\x1b[" + inspect.colors[e][1] + "m" : r;
+                            return e ? "[" + inspect.colors[e][0] + "m" + r + "[" + inspect.colors[e][1] + "m" : r;
```

Comment:
Seems like a wrong change?

Reply:
In original code it is `ESC` character, so we keep them as is for non minified output

Reply:
Oh, I see it's not rendered correctly in diff views.
I found that it's identical in file view, thanks!